sdk/storage/azblob/internal/generated/autorest.md

```diff
@@ -301,3 +301,13 @@ directive:
     return $.
       replace(/IncrementalCopyOfEralierVersionSnapshotNotAllowed/g, "IncrementalCopyOfEarlierVersionSnapshotNotAllowed");
 ```
+
+### Fix up x-ms-content-crc64 header response name
+
+``` yaml
+directive:
+- from: swagger-document
+  where: $.x-ms-paths.*.*.responses.*.headers.x-ms-content-crc64
+  transform: >
+    $["x-ms-client-name"] = "ContentCRC64"
```

Comment:
Any particular reason so make this change? just trying to understand the rationale behind this.

Reply:
The short answer is the name is ugly.

For the long answer, MS-specific headers have the `x-ms-` prefix.  By convention, we remove this from the names as the prefix `XMS` doesn't add any value to the SDK.